dbt/adapters/bigquery/connections.py

```diff
@@ -519,10 +519,7 @@ def get_bq_table(self, database, schema, identifier) -> Table:
         # backwards compatibility: fill in with defaults if not specified
         database = database or conn.credentials.database
         schema = schema or conn.credentials.schema
-        return client.get_table(
-            table=self.table_ref(database, schema, identifier),
-            retry=self._retry.create_reopen_with_deadline(conn),
```

Comment:
This was added during the retry factory work in #1395 but should have remained unset to continue using the default. No test changes are needed as this is reverting a change.